workflow.qmd

```diff
@@ -0,0 +1,41 @@
+---
+title: "U.S. Greenhouse Gas Center: Data Workflow Documentation"
+---
+
+Welcome to the [U.S. Greenhouse Gas (GHG) Center](https://earth.gov/ghgcenter) data workflow documentation. View data flow diagrams to discover the journey of each dataset from acquisition into the US GHG Center.
+
+Data flow diagrams are grouped topically and labeled by dataset name. The dataset product type (model output, satellite observation, etc.) is noted next to each. Click on a dataset name to view the data flow diagram for that dataset, which summarizes the process followed to bring the dataset into the US GHG Center.
+
+Join us in our mission to make data-driven environmental solutions accessible.
+Explore, analyze, and make a difference with the US GHG Center.
+
+[Back to Data Catalog](https://earth.gov/ghgcenter/data-catalog)
+
+## Gridded Anthropogenic Greenhouse Gas Emissions
+1. [OCO-2 MIP Top-Down CO₂ Budgets](data_workflow/oco2-mip-co2budget-yeargrid-v1_Data_Flow.qmd) - Model Output
```

Comment:
Consider adding 'data flow diagram' to each. E.g. 
"OCO-2 MIP Top-Down CO₂ Budgets Data Flow Diagram"

Reply:
Also remove "- Model Output"